plotly 0.8.1 seems not work with kaleido 0.3.0

Is there anything wrong with this?

**Envrionment:**
Toolchain: stable-x86_64-pc-windows-msvc
Rust/Cargo: 1.64.0

**Steps to reproduce:**
  cargo init app_name
  cd app_name
  cargo add plotly
  cargo add plotly --features kaleido
  cargo check

**Output:**
....
```
   Compiling plotly_kaleido v0.3.0
    Checking plotly v0.8.1
error[E0308]: mismatched types
   --> C:\Users\mwim\.cargo\registry\src\github.com-1ecc6299db9ec823\plotly-0.8.1\src\plot.rs:385:17
    |
383 |             .save(
    |              ---- arguments to this function are incorrect
384 |                 filename.as_ref(),
385 |                 &serde_json::to_value(self).unwrap(),
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `str`, found enum `Value`
    |
    = note: expected reference `&str`
               found reference `&Value`
note: associated function defined here
   --> C:\Users\mwim\.cargo\registry\src\github.com-1ecc6299db9ec823\plotly_kaleido-0.3.0\src\lib.rs:124:12
    |
124 |     pub fn save(
    |            ^^^^

For more information about this error, try `rustc --explain E0308`.
error: could not compile `plotly` due to previous error
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 ## [0.8.3] - 2022-11-04
 ### Fixed
 - [[#122](https://github.com/igiagkiozis/plotly/pull/122)] Compilation error for the `wasm` feature.
+- [[#123](https://github.com/igiagkiozis/plotly/pull/123)] Compilation error for the `plotly_kaleido` feature.
 
 ## [0.8.2] - 2022-11-03
 ### Added

--- a/README.md
+++ b/README.md
@@ -55,7 +55,7 @@ Add this to your `Cargo.toml`:
 
 ```toml
 [dependencies]
-plotly = "0.8.2"
+plotly = "0.8.3"
 ```
 
 ## Exporting an Interactive Plot
@@ -97,7 +97,7 @@ To save a plot as a static image, the `kaleido` feature is required:
 # Cargo.toml
 
 [dependencies]
-plotly = { version = "0.8.2", features = ["kaleido"] }
+plotly = { version = "0.8.3", features = ["kaleido"] }
 ```
 
 With this feature enabled, plots can be saved as any of `png`, `jpeg`, `webp`, `svg`, `pdf` and `eps`. Note that the plot will be a static image, i.e. they will be non-interactive.
@@ -124,7 +124,7 @@ Using `Plotly.rs` in a Wasm-based frontend framework is possible by enabling the
 # Cargo.toml
 
 [dependencies]
-plotly = { version = "0.8.2", features = ["wasm"] }
+plotly = { version = "0.8.3", features = ["wasm"] }
 ```
 
 First, make sure that you have the Plotly JavaScript library in your base HTML template:

--- a/docs/book/src/getting_started.md
+++ b/docs/book/src/getting_started.md
@@ -22,7 +22,7 @@ To start using [plotly.rs](https://github.com/igiagkiozis/plotly) in your projec
 
 ```toml
 [dependencies]
-plotly = "0.8.2"
+plotly = "0.8.3"
 ```
 
 [Plotly.rs](https://github.com/igiagkiozis/plotly) is ultimately a thin wrapper around the `plotly.js` library. The main job of this library is to provide `structs` and `enums` which get serialized to `json` and passed to the `plotly.js` library to actually do the heavy lifting. As such, if you are familiar with `plotly.js` or its derivatives (e.g. the equivalent Python library), then you should find [`plotly.rs`](https://github.com/igiagkiozis/plotly) intuitive to use.
@@ -97,7 +97,7 @@ To add the ability to save plots in the following formats: png, jpeg, webp, svg,
 
 ```toml
 [dependencies]
-plotly = { version = "0.8.2", features = ["kaleido"] }
+plotly = { version = "0.8.3", features = ["kaleido"] }
 ```
 
 ## WebAssembly Support

--- a/plotly/Cargo.toml
+++ b/plotly/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "plotly"
-version = "0.8.2"
+version = "0.8.3"
 description = "A plotting library powered by Plotly.js"
 authors = ["Ioannis Giagkiozis <i.giagkiozis@gmail.com>"]
 license = "MIT"
@@ -26,8 +26,8 @@ erased-serde = "0.3"
 getrandom = { version = "0.2", features = ["js"], optional = true }
 image = { version = "0.24.2", optional = true }
 js-sys = { version = "0.3", optional = true }
-plotly_derive = { version = "0.8.2", path = "../plotly_derive" }
-plotly_kaleido = { version = "0.3.0", path = "../plotly_kaleido", optional = true }
+plotly_derive = { version = "0.8.3", path = "../plotly_derive" }
+plotly_kaleido = { version = "0.8.3", path = "../plotly_kaleido", optional = true }
 ndarray = { version = "0.15.4", optional = true }
 once_cell = "1"
 serde = { version = "1.0.132", features = ["derive"] }
@@ -44,5 +44,5 @@ image = "0.24.4"
 itertools = "0.10.3"
 itertools-num = "0.1.3"
 ndarray = "0.15.4"
-plotly_kaleido = { version = "0.3.0", path = "../plotly_kaleido" }
+plotly_kaleido = { version = "0.8.3", path = "../plotly_kaleido" }
 rand_distr = "0.4"

--- a/plotly_derive/Cargo.toml
+++ b/plotly_derive/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "plotly_derive"
-version = "0.8.2"
+version = "0.8.3"
 description = "Internal proc macro crate for Plotly-rs."
 authors = ["Ioannis Giagkiozis <i.giagkiozis@gmail.com>"]
 license = "MIT"

--- a/plotly_kaleido/Cargo.toml
+++ b/plotly_kaleido/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "plotly_kaleido"
-version = "0.3.0"
+version = "0.8.3"
 description = "Additional output format support for plotly using Kaleido"
 authors = ["Ioannis Giagkiozis <i.giagkiozis@gmail.com>"]
 license = "MIT"

